src/locales/en.js

```diff
@@ -74,6 +74,7 @@ export default {
   '_ol_kit.PopupDefaultPage.details': 'Details',
   // Popup Actions
   '_ol_kit.PopupActionWkt.copyToClipboard': 'Copy Geometry as Coordinates',
+  '_ol_kit.PopupActionGoogleMaps.navigateGoogleMaps': 'Navigate here with Google Maps',
```

Comment:
'View location in Google Maps'

Reply:
That's not what this does, this literally starts navigation mode, but I can be a team player.